frontend/src/component/feature/StrategyTypes/FlexibleStrategy/StickinessSelect/StickinessSelect.tsx

```diff
@@ -51,7 +51,11 @@ export const StickinessSelect = ({
             disabled={!editable}
             data-testid={dataTestId}
             onChange={onChange}
-            style={{ width: 'inherit', minWidth: '100%' }}
+            style={{
+                width: 'inherit',
+                minWidth: '100%',
+                marginBottom: '16px',
```

Comment:
This could/should be expressed with `theme.spacing` for consistency

Reply:
sx is not supported on this component